lte/gateway/fabfile.py

```diff
@@ -49,6 +49,7 @@
     fab dev package upload_to_aws
 """
 
+IP_ADDRESS = "192.168.60.142"
```

Comment:
Nit: Might be preferable to indicate in the name what it's the IP of, e.g. `GATEWAY_IP_ADDRESS`